### DISCUSSION TITLE
Ikea motion sensors - old E1525/E1745 vs. new Vallhorn E2134

### DISCUSSION THREAD

**Discussion by Elephantoplasty** [Koenkk/zigbee2mqtt#23580] [Devices]
Hi All,

I've tried quite a few zigbee motion sensors, and found the old Ikea E1525/E1745 to be the fastest responder of them all, and very reliable.   Second best are the Philips Hue ones in terms of response times.

I needed some more, and it appears the old one has been replaced by the Vallhorn.  I hoped that the new Vallhorn would be comparable to the old one in terms of response time, but it is sadly a deal slower.

I ran a side-to-side comparison of the two with a light in my walk-in wardrobe, and whereas the old one has the light on before you have got through the doorway, with the new one there is a couple of seconds delay - you end up pausing in the wardrobe waiting for the light to come on.

Anyone else find the new one a disappointment?  Beyond a hopefully longer battery life, I can't see any advantages to it at all other than it reports illuminance which I find of no real use.  It is big, obtrusive and slow.

Makes me wonder why they replaced the old one.....

**Comment by jnswb** [comment]
I only have E2134 models from Ikea so can’t compare. But the E2134 is kind of random in its speed. Sometimes it reacts really slowly (several seconds delay) and sometimes fast.

I have one Lidl HG07310 and don’t see similar (random) delays.

**Comment by Elephantoplasty** [comment]
I've fiddled with the positioning of mine a bit more to enable a fairer comparison to the old one.   At times it is about as fast, but like you, still the odd occasion when there is a significant lag.

**Comment by nachtbraker** [comment]
Same here, the Valhorn reacts really slow compared to the E1525/E1745.

Any advice for an affordable, fast reacting motion sensor?

**Reply by djurny** [reply]
Hi there,
I have several [Sonoff SNZB-03](https://www.zigbee2mqtt.io/devices/SNZB-03.html) sensors which work OK. The detection time is short, it usually takes a second or so for the sensor to flash (indicating it spotted movement). Quite similar to the [Ikea TRADFRI E1525/E1745](https://www.zigbee2mqtt.io/devices/E1525_E1745.html) sensors.

The cooldown time for [Sonoff SNZB-03](https://www.zigbee2mqtt.io/devices/SNZB-03.html) sensors is around 1 minute (i.i.r.c.) compared to 3 minutes of the [Ikea TRADFRI E1525/E1745](https://www.zigbee2mqtt.io/devices/E1525_E1745.html) sensors, compared to the approx. 30 seconds of the [Ikea VALLHORN E2134](https://www.zigbee2mqtt.io/devices/E2134.html) sensors.

The [Sonoff SNZB-03](https://www.zigbee2mqtt.io/devices/SNZB-03.html) sensors suffer from the same troubles as the [Ikea TRADFRI E1525/E1745](https://www.zigbee2mqtt.io/devices/E1525_E1745.html), stuck occupancy as they do not report occupancy every now and then. Missed an event? Stuck. The [Ikea VALLHORN E2134](https://www.zigbee2mqtt.io/devices/E2134.html) sensors have not been in such a stuck situation yet.

Hope that helps,
Groetjes,

**Reply by nachtbraker** [reply]
Thanks, I'll try the Sonoff!

**Comment by malaakso** [comment]
I wonder if this is due to the way the occupancy is reported. As far as I understand, Vallhorn has the new occypancy sensing cluster, where occupancy is reflected in an attribute that is reported to the coordinator. But it also has the old generic on/off cluster (like old Tradfri sensor) which can send "on" command to a bound target (coordinator or lamp) when motion is detected.
